src/plugins/opensearch_dashboards_react/public/table_list_view/table_list_view.tsx

```diff
@@ -413,10 +417,10 @@ class TableListView extends React.Component<TableListViewProps, TableListViewSta
   renderTable() {
     const selection = this.props.deleteItems
       ? {
-          onSelectionChange: (obj: Item[]) => {
+          onSelectionChange: (obj: Object[]) => {
```

Comment:
Is this type used correctly? `Object[]`?

Reply:
thanks, changed it back to Item[]